frontend/src/views/Game.tsx

```diff
@@ -147,8 +148,28 @@ function GamePage() {
     setTimeUp(newGame.gameTimer.timeUp);
   };
 
+  const createCodeLanguageArray = () => {
+    while (currentLanguage.length < problems.length) {
+      currentLanguage.push(Language.Python);
+    }
+
+    while (currentCode.length < problems.length) {
+      currentCode.push('');
+    }
+  };
+
+  createCodeLanguageArray();
```

Comment:
If I understood correctly, I'm ignoring this - I do have comments so let me know when / if you want me to add those (could be here or on a new PR).

Reply:
Umm, yeah, this code is necessary to switch between problems, so we can talk about this in pair programming later today.